tests/logictest/suites/crdb/join

```diff
@@ -0,0 +1,899 @@
+statement ok
+drop table if exists onecolumn;
+
+statement ok
+CREATE TABLE onecolumn (x INT NULL);
```

Comment:
FYI, the bug https://github.com/datafuselabs/databend/issues/7149 was fixed, maybe you can add it @ZeaLoVe